.github/workflows/docker-develop.yml

```
@@ -14,6 +14,8 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v3.3.0
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

```suggestion
        uses: docker/setup-buildx-action@v2.3.0
```

何もしていないのに急に動かなくなった、という事態を防ぐためにバージョンを明示的に指定すると良さそうです。